packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1073,6 +1073,43 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     goBallistic(0.0);
   }
 
+  void pointerScroll(double delta) {
+    assert(delta != 0.0);
+
+    delta = -delta;
+
+    goIdle();
+    updateUserScrollDirection(
+        delta > 0.0 ? ScrollDirection.forward : ScrollDirection.reverse);
+    if (_innerPositions.isEmpty) {
+      _outerPosition!.applyClampedDragUpdate(delta);
```

Comment:
A pointer scroll is not a drag event, and should not be implemented using drag logic.

Reply:
This might be ok, `applyClampedDragUpdate` just may not be an appropriate name anymore, maybe `applyClampedPositionUpdate`?
It follows the same conventions of calculating the target pixels and updating the scroll position just with extra logic to reconcile between the two positions. I don't see that it has any drag-specific behavior, nor is it much different from the implementation of `ScrollPositionWithSingleContext.pointerScroll` below.

Reply:
I thought about changing the name, but [applyClampedDragUpdate] considered [ScrollPhysics], and I ended up creating an [applyClampedPointerSignalUpdate] to fix it.